### PR TITLE
Issue 2501 upgrade 0.0.8

### PR DESCRIPTION
**Pull request by ozten**
Currently, the `dev` branch creates and uses `client.json` which is actually a JavaScript file, not a proper JSON file.

This is fixed in i18n-abide 0.0.8.

There are a couple tweaks to tests, file names, etc so that the code works with an actual JavaScript file on the client side as well as proper JSON files on the server side for localization.

### CODE REVIEW

File: lockdown.json [hunk 1]
```diff
@@ -146,14 +164,17 @@
...
   },
   "gobbledygook": {
     "0.0.3": "437bb23d3ade04dd26b49e7c21e4026c6b086234"
   },
   "graceful-fs": {
     "1.1.14": "07078db5f6377f6321fceaaedf497de124dc9465",
-    "1.2.1": "b35cc6e623576fc2a278cba12c00dda6a1758d2d"
+    "1.2.1": "b35cc6e623576fc2a278cba12c00dda6a1758d2d",
+    "1.2.3": "*",
+    "2.0.0": "*"
```
Comment (seanmonstar) [inline lockdown.json]: I know its a big PITA, but we can't have these stars in the lockdown.

File: lockdown.json [hunk 2]
```diff
@@ -165,26 +186,37 @@
     "0.1.1": "d9d82735a5f85f950761ac3909ba9485ec0af4f1"
   },
   "i18n-abide": {
-    "0.0.8-beta4": "b106bdaa156f90652e04b75ef6a1079c271a1b45"
+    "0.0.8": "a7610465be083a21e1c2f52e2fd78503ee90dd29"
```
Comment (seanmonstar) [inline lockdown.json:189]: Excite!

File: lockdown.json [hunk 3]
```diff
@@ -146,14 +164,17 @@
     "1.1.1": "1f89623453123ef9623956e264c60bf4c3cf5ccf"
   },
   "glob": {
-    "3.2.1": "57af70ec73ba2323bfe3f29a067765db64c5d758"
+    "3.2.1": "57af70ec73ba2323bfe3f29a067765db64c5d758",
+    "3.2.3": "e313eeb249c7affaa5c475286b0e115b59839467"
```
Comment (ozten) [inline lockdown.json:168]: Done.

File: resources/static/dialog/js/modules/add_email.js [hunk 4]
```diff
@@ -43,11 +43,16 @@ BrowserID.Modules.AddEmail = (function() {
...
       dom.setAttr(EMAIL_SELECTOR, 'disabled', 'disabled');
       dom.addClass(BODY_SELECTOR, SUBMIT_DISABLED_CLASS);
+
       dialogHelpers.addEmail.call(self, email, function removeHint(status) {
+        hideHint("addressInfo");
+
+        // !status means there was a problem adding the email. Let the user
+        // modify the email address and try to re-add.
         if (!status) {
-          hideHint("addressInfo");
```
Comment (jaredhirsch) [inline resources/static/dialog/js/modules/add_email.js]: can you rebase against dev? this diff doesn't belong here, it was already added by 00a7423c

File: resources/static/test/cases/dialog/js/modules/add_email.js [hunk 5]
```diff
@@ -65,12 +66,16 @@
...
+      stagedEmail = info.email;
     });
 
-    controller.addEmail();
+    controller.addEmail(function() {
+      equal(stagedEmail, normalizedEmail, "stage_email called with correct email");
+      testHelpers.testNotVisible(".addressInfo");
+      start();
+    });
   }
```
Comment (jaredhirsch) [inline resources/static/test/cases/dialog/js/modules/add_email.js:79]: ditto for this file, these changes don't belong in this commit, they were already added to dev via 00a7423c

**Comment by jaredhirsch** [comment]
@ozten does this change need to go in the train that's branching tomorrow? If so, I'm happy to drive it over the finish line once you rebase against dev, just ping me when it's updated.

**Comment by ozten** [comment]
yes, reviewing comments. Thanks for your help @6a68

**Comment by ozten** [comment]
rebasified.

**Comment by jaredhirsch** [comment]
@ozten testing locally now

**Comment by jaredhirsch** [comment]
unit tests passing for me locally, merging in.

**Comment by ozten** [comment]
FWIW: Travis has this as a failing build. I'm green locally. I've restarted the build several times. So it goes.

**Comment by jrgm** [comment]
I restarted a single failing job (of 6) and it is now all green.

**Comment by ozten** [comment]
Proof - Travis obeys jrgm!
